pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -162,3 +162,29 @@ class EmuFreeBackend(PasqalEmulator):
 
     emulator = pasqal_cloud.EmulatorType.EMU_FREE
     default_config = DEFAULT_CONFIG_EMU_FREE
+
+
+class EmuMpsBackend(RemoteBackend):
```

Comment:
Did you try making this inherit from `EmulatorBackend` too?
```suggestion
class EmuMpsBackend(RemoteBackend, EmulatorBackend):
```

Reply:
Shame...